bin/www

```diff
@@ -44,14 +56,22 @@ app.set('port', port);
  * Create HTTP server.
  */
 
-const httpsOptions = {
-    key: (environment === 'production') ? null : fs.readFileSync( './localhost.key' ),
-    cert: (environment === 'production') ? null : fs.readFileSync( './localhost.cert' ),
-    requestCert: false,
-    rejectUnauthorized: false
-};
+function configureServer() {
+    if (environment === 'production' || (marianaApiHost && isHostHttp)) {
```

Comment:
is this right? it looks like if the environment is production then here we'll stop and return an http server. this seems to contradict some logic below, for example `key: (environment === 'production') ? null : fs.readFileSync( './localhost.key' )` at this point `environment === 'production'` is always false.

Reply:
G-G-G-G-Great point, chopped it up with Welch to make sure we're not missing anything here and we're not. I adjusted the `httpsOptions` for `key` and `cert` so they either have values or are null.